rs/sns/governance/src/topics.rs

```diff
@@ -252,12 +253,279 @@ impl pb::Governance {
             })
             .collect()
     }
+
+    pub fn get_topic_for_action(
```

Comment:
Putting an implementation directly on the protobuf seems like a bad pattern.  I see that it already exists here, but it would be better to put it on the Governance struct instead, as you could then more easily change the method of data storage (such as when you move into stable memory)

Reply:
I agree with the concern. However, the `get_topic_for_action` function (which was added before this PR) is called from `validate_and_render_action`, and for some reason we don't have an instance of Governance available there, only `env` and `governance_proto`. So avoiding all implementation on `pb::Governance` requires a bigger refactoring, which I suggest not doing in the scope of this PR. 

However, I did more `get_topic_for_action` to `Governance`, as this new function is called only from `make_proposal`, which itself is a function of `Governance`, not `pb::Governance`.

Reply:
This was quite a rabbit hole jump, but now `get_topic_for_action` is moved from pb::Governance to Governance.